src/odemis/driver/static.py

```diff
@@ -110,7 +111,21 @@ def __init__(self, name, role, mag, mag_choices=None, na=0.95, ri=1,
         if rotation is not None:
             self.rotation = model.FloatContinuous(rotation, (0, 2 * math.pi),
                                                   unit="rad")
-
+        if configurations is not None:
+            self._configurations = configurations
+            conf = list(configurations.keys())[0]
```

Comment:
Ideally, that initial conf would be picked based on the current values of the VAs.